src/mainwindow.cpp

```diff
@@ -210,6 +220,31 @@ MainWindow::MainWindow(QWidget* parent)
            "E.g. show std::string instead of std::basic_string<char, ...>"));
     connect(prettifySymbolsAction, &QAction::toggled, Settings::instance(), &Settings::setPrettifySymbols);
 
+    auto* collapseTemplatesAction = ui->viewMenu->addAction(tr("Collapse Templates"));
+    collapseTemplatesAction->setCheckable(true);
+    collapseTemplatesAction->setChecked(Settings::instance()->collapseTemplates());
+    collapseTemplatesAction->setToolTip(
+        tr("Collapse complex templates to simpler ones E.g. QHash<...> instead of QHash<QString, QVector<QString>>"));
```

Comment:
prevent the <> from being detected as html, better format this as:

```
tr("<qt>Collapse ...E.g.: <tt>QHash&lt;...&gt;</tt> instead of <tt>QHash&lt;QString,...</tt>.</qt>");
```